Create advertising_indication.yml

pdfext opened the pull request:
#Profil OSM contributor pour renseigner les dispositifs publicitaires de type 'pré-enseigne'.

**Préenseigne publicitaire** : **seul** type de publicité autorisé en France **hors-agglomération**.
Dispositif publicitaire très réglementé en France, pour présignaliser une activité, avec en général une indication de direction et/ou de distance. Se différencie de l'enseigne par le fait qu'elle se situe en dehors du fonds où s'exerce l'activité. La taille est réglementée : 1,5 m de large sur 1 m de haut. En général au bord des routes rurales.
Depuis la loi ENE (12/07/2010) et après un délai de 5 ans de mise en conformité, donc depuis le 15 juillet 2015, seuls 4 types d'activités peuvent encore utiliser ces dispositifs :
 *Fabrication et vente de produits du terroir par des entreprises locales.
 *Activité culturelle (voir lesquelles).
 *Monuments historiques ouverts à la visite.
 *Certaines activités exceptionnelles temporaires.
La présignalisation hors-agglomération pour
 *Station service
 *Garage
 *Hôtel
 *Restaurant
 *Activité en retrait de la voie publique
 *Magasin
 *Fast-food
 *Supermarché
   etc. est strictement **interdite** sur ces dispositifs depuis le 13 juillet 2015. Ces genres d'activité doivent se présignaliser maintenant avec de la SIL (Signalisation d'Information Locale).

pdfext committed:
Create advertising_indication.yml

**Préenseigne publicitaire** : **seul** type de publicité autorisé en France **hors-agglomération**.
Dispositif publicitaire très réglementé en France, pour présignaliser une activité, avec en général une indication
de direction et/ou de distance. Se différencie de l'enseigne par le fait qu'elle se situe en dehors du fonds où
s'exerce l'activité. La taille est réglementée : 1,5 m de large sur 1 m de haut. En général au bord des routes rurales.
Depuis la loi ENE (12/07/2010) et après un délai de 5 ans de mise en conformité, donc depuis le 15 juillet 2015,
seuls 4 types d'activités peuvent encore utiliser ces dispositifs :
 *Fabrication et vente de produits du terroir par des entreprises locales.
 *Activité culturelle (voir lesquelles).
 *Monuments historiques ouverts à la visite.
 *Certaines activités exceptionnelles temporaires.
La présignalisation hors-agglomération pour
 *Station service
 *Garage
 *Hôtel
 *Restaurant
 *Activité en retrait de la voie publique
 *Magasin
 *Fast-food
 *Supermarché
   etc. est strictement **interdite** sur ces dispositifs depuis le 13 juillet 2015. Ces genres d'activité doivent
   se présignaliser maintenant avec de la SIL (Signalisation d'Information Locale).

loicortola commented on presets/advertising_indication.yml:
```diff
@@ -0,0 +1,114 @@
+author: PaDe
+description: 
+  default: "Pour participer à l'inventaire des préenseignes publicitaires en France, qui sont presque à 100% illégales."
+  en: "To participate in inventorying advertising indication boards in France, which are almost 100% unlawfull."
```
Si possible, merci de rester objectif dans la description, sans jugement de valeur ("qui sont presque 100% illégales")

Proposition: remplacer la description anglaise par celle-ci:
"To participate in the creation of the inventory for advertising indicator boards"

loicortola replied:
Est-il possible de généraliser ce preset pour les boards hors de France?

loicortola commented on presets/advertising_indication.yml:
```diff
@@ -0,0 +1,114 @@
+            action: 
+              key: "legal_status:FR"
+              type: SET_TAG_VALUE
+              value: illégal
```
valeur du tag pourrait être internationalisée aussi (creation d'un legal_status ?)

loicortola replied:
Cette contrainte place un legal_status de façon systématique lorsque la description est "autres"... Est-ce vraiment le comportement souhaité par la communauté?

loicortola commented on presets/advertising_indication.yml:
```diff
@@ -0,0 +1,114 @@
+              type: TAG
+        description: 
+          default: "Une préenseigne publicitaire, dispositif publicitaire hors-agglomération, pour présignaliser une activité."
+          en: "An indication board, found along a rural road, to advertise proximity of a craft/commercial/industrial activity. Rural advertising board."
```
An indication board found along a rural road, which advertises the proximity of a craft/commercial/industrial activity.

pdfext committed:
To take into account the review

I changed the wordings to follow what you suggested. It is unfortunately not possible to make this preset usable in any country. French rules as far as outdoor advertising is concerned are very specific, and quite complex too. This is very likely that the rules are not the same in other countries. Therefore this preset is only usable in France, which is already a lot. But This preset is usable by english people as it is localized in english. 

Il n'y a que 4 types d'activités qui peuvent être annoncées pas des préenseignes comme c'est mentionné dans le descriptif. Tout autre type d'activité rend la préenseigne illégale comme c'est expliqué par le descriptif (Sans être expert du Code de l'environnement, je connais un peu la réglementation sur la publicité extérieure en France). 
Voir http://www.fidal-avocats-leblog.com/2015/10/publicite-exterieure-place-au-demantelement-pre-enseignes-derogatoires-devenues-illegales/
Voir http://lentreprise.lexpress.fr/marketing-vente/promotion-communication/panneaux-publicitaires-tout-doit-disparaitre_1687820.html
Voir http://www.ain-pro.com/index.php?id=actualite_2_1_1&art=218&itemracine=ITEMRACINE_ACTUALITE&global_id_item=GLOBALIDITEM_ACTUALITE&HideMenu=1

Il n'y a pas de tag actuellement pour ceci dans OSM, et c'est pour cela que j'ai moi-même forgé "legal_status:FR" qui est très récent et qui n'a pas encore de page wiki.
